dev/io.openliberty.persistence.3.2.thirdparty/resources/org/eclipse/persistence/internal/localization/i18n/LoggingLocalizationResource.nlsprops

```diff
@@ -169,6 +169,14 @@ elements={0}{1} elements
 
 enable_batch_operations_as_true_not_supported=WLS native CMP setting "enable-batch-operations-as-true" is not directly supported in EclipseLink CMP. See migration doc for details.
 
+encryptor_script_description=This application internally decrypt old encrypted password used by some previous version EclipseLink and encrypt it by latest algorithm.
```

Comment:
The message appears to contain unwanted words. How about...

This application has an old encrypted password that was used by a previous version of EclipseLink. Reencrypt it with the latest algorithm.